custom/icds_reports/reports/fact_sheets.py

```diff
@@ -497,41 +497,72 @@ def new_table_config(self):
                                 'slug': 'four',
                                 'average': [],
 
-                            },
-                            {
-                                'data_source': 'AggAWCMonthlyDataSource',
-                                'header': 'Adolescent girls (11-14 years)',
-                                'slug': 'cases_person_adolescent_girls_11_14_all',
-                                'average': [],
-
-                            },
-                            {
-                                'data_source': 'AggAWCMonthlyDataSource',
-                                'header': 'Adolescent girls (15-18 years)',
-                                'slug': 'cases_person_adolescent_girls_15_18_all',
-                                'average': [],
-
-                            },
-                            {
-                                'data_source': 'AggAWCMonthlyDataSource',
-                                'header': 'Adolescent girls (11-14 years)  enrolled for Anganwadi Services',
-                                'slug': 'cases_person_adolescent_girls_11_14',
-                                'average': [],
-
-                            },
-                            {
-                                'data_source': 'AggAWCMonthlyDataSource',
-                                'header': 'Adolescent girls (15-18 years)  enrolled for Anganwadi Services',
-                                'slug': 'cases_person_adolescent_girls_15_18',
-                                'average': [],
-
                             }
                         ]
                     },
                 ]
             }
         ]
 
+        if not self.beta:
```

Comment:
This changes fact sheet
![fact_sheet_@](https://user-images.githubusercontent.com/21310920/71279615-34179180-237f-11ea-8f0c-72280424b79a.png)